### ISSUE TITLE
Cannot Compile For Windows

### ISSUE THREAD

**Issue by rtablada** [EmberGrep/sqlite-ez#1]
Right now launching on windows fails since `requireNode('sqlite3')` is throwing an error so the `application` controller module fails to load.

I'm not sure if anyone has experience with `sqlite3` + Windows and if you can help out here?

**Comment by elpiel** [comment]
It's not only in Windows, I am under Ubuntu and I have this error too.

**Comment by jkeen** [comment]
Yep, I came here trying to figure out how to get sqlite3 working with ember-electron, and turns out it doesn't work in this app either.
